build.sbt

```diff
@@ -17,7 +17,6 @@ ThisBuild / scalaVersion := scala3_3_3
 lazy val microservice = Project(appName, file("."))
```

Comment:
Do you need to remove the targetJvm from the default bindings?

Reply:
It's recommended to remove the setting completely by the platform in their blog post about it.